Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -1461,3 +1460,562 @@ theorem hasFPowerSeriesAt_iff' :
 #align has_fpower_series_at_iff' hasFPowerSeriesAt_iff'
 
 end
+
+
+section FiniteFPowerSeries
```

Comment:
I think this warrants starting a new file. The current file is already quite a bit > 1000 lines, and you are adding another >500.

Reply:
> I think this warrants starting a new file. The current file is already quite a bit > 1000 lines, and you are adding another >500.

Okay, so make a file called something like `Polynomial` and put all the modifications from the other two files in it ?
(By the way, I didn't do stability by composition, because there's not much I can add except an estimate for the degree of the composition, and that seemed like a lot of work for something that I'm not even sure we need yet.)

Reply:
I implemented your suggestions and put the "finite powers series" stuff in a file `Polynomial.lean`, but left the things about derivatives in the same file as before (since they happen in a different part of the mathlib hierarchy). Apparently this has created a conflict ? I see an ominous message "This branch has conflicts that must be resolved", but I don't know what the conflict is.

Reply:
Ok oops, the file `Polynomial.lean` already existed in that directory. I'm surprised that vscode didn't protest when I tried to create a new file with that name.

Reply:
I *think* I got rid of the merge conflict, but now I get new CI fails, and I frankly have no idea what is going on.